pkg/kubectl/resource_printer.go

```diff
@@ -599,6 +615,11 @@ func printPod(pod *api.Pod, w io.Writer, options PrintOptions) error {
 func printPodBase(pod *api.Pod, w io.Writer, options PrintOptions) error {
 	name := pod.Name
 	namespace := pod.Namespace
+	kind := options.KindName
```

Comment:
Hm, I am receiving an error when trying to cast an object to `*unversioned.TypeMeta` any time I pass it to the above function. Also, any time I print `obj.GroupVersionKind().Kind`, I get an empty string.